net/libfab/libfab.c

```diff
@@ -2790,6 +2708,27 @@ static int libfab_domain_params_get(struct m0_fab__ndom *fab_ndom)
 	return M0_RC(0);
 }
 
+static int libfab_buf_dom_dereg(struct m0_fab__buf *fbp)
+{
+	struct m0_fab__ndom *nd = fbp->fb_nb->nb_dom->nd_xprt_private;
+	int                  i;
+	int                  ret = 0;
+
+	for (i = 0; i < nd->fnd_seg_nr; i++) {
+		if (fbp->fb_mr.bm_mr[i] != NULL) {
+			ret = fi_close(&fbp->fb_mr.bm_mr[i]->fid);
+			if (ret != 0)
+				break;
```

Comment:
Errors of all external functions (including fi_*()) should be logged.

Reply:
Added trace log